### DISCUSSION TITLE
Option to hide the light/dark switch icon?

### DISCUSSION THREAD

**Discussion by afranke** [pradyunsg/furo#774] [Questions]
Is there an option to not show the dark/light theme switcher in the top right corner? I read the docs and expected something in `html_theme_options`, but couldn’t find it.

**Comment by afranke** [comment]
More specifically, I was expecting `top_of_page_button` to be the option for that, but the docs clearly say that `edit` and `None` are the only supported values. :frowning_face:

**Comment by pradyunsg** [comment]
> Is there an option to not show the dark/light theme switcher in the top right corner?

No, there is not. Why do you want this?

**Reply by overfloddedwithCss** [reply]
I can only speak for myself, but in my use of Furo it would be more than helpful to have this option. My documentation is not meant to be used in dark mode, it just doesn't look good. There are a lot of images in there and the contrast between the black background and the images is too strong.
